src/Tickets/Commerce/Gateways/Free/Gateway.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Tickets Commerce: Free Gateway.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Gateways\Free
+ */
+
+namespace TEC\Tickets\Commerce\Gateways\Free;
+
+use TEC\Tickets\Commerce\Cart;
+use TEC\Tickets\Commerce\Gateways\Contracts\Abstract_Gateway;
+use TEC\Tickets\Commerce\Module;
+use Tribe__Template as Template;
+
+/**
+ * Class Free Gateway.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Gateways\Free
+ */
+class Gateway extends Abstract_Gateway {
+	/**
+	 * The Gateway key.
+	 *
+	 * @since TBD
+	 *
+	 * @var string $key The Gateway key.
+	 */
+	protected static $key = 'free';
+	
+	/**
+	 * @inheritDoc
+	 */
+	public static function get_label() {
+		return '';
+	}
+	
+	/**
+	 * @inheritDoc
+	 */
+	public static function is_enabled(): bool {
+		return true;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function is_connected() {
+		return true;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function is_active() {
```

Comment:
```
	public static function is_active() {
	      return tribe( Module::class )->is_checkout_page() && static::should_show();
```
A bit picky here, but if I were to instantiate this and want to check whether this module is `active` but I wasn't on the checkout page, it would always return `false` right? Perhaps rename or move conditional checks a bit?

Reply:
We have 3 things here:

[is_enabled](https://github.com/the-events-calendar/event-tickets/blob/a40a7e3d262771d5fa54438c732936d8e47b84e9/src/Tickets/Commerce/Gateways/Contracts/Abstract_Gateway.php#L271) :  If the gateway is enabled from settings or not.

[is_connected](https://github.com/the-events-calendar/event-tickets/blob/a40a7e3d262771d5fa54438c732936d8e47b84e9/src/Tickets/Commerce/Gateways/Contracts/Abstract_Gateway.php#L113): If the gateway is configured properly.

[is_active](https://github.com/the-events-calendar/event-tickets/blob/a40a7e3d262771d5fa54438c732936d8e47b84e9/src/Tickets/Commerce/Gateways/Contracts/Abstract_Gateway.php#L125): Means ready for Checkout with both of the above being true.

For this free gateway `is_active` is not dependent on any 3rd party, so we are just checking the checkout page.

So,  with instantiate should be looking for the is_enabled or is_connected value. 

BTW [both](https://github.com/the-events-calendar/event-tickets/blob/b80963969dbc84d23e619146932fd99e0902ee12/src/Tickets/Commerce/Gateways/Free/Gateway.php#L48-L57) are true for this one as we don't allow users to turn this off with any setting.

I hope this clears things up.